web-frontend/src/assets/util.js

```
@@ -0,0 +1,587 @@
+					side: null,
+
+				// Target element for "class".
+					target: $this,
+
+				// Class to toggle.
+					visibleClass: 'visible'
+
+			}, userConfig);
+
+			// Expand "target" if it's not a jQuery object already.
+				if (typeof config.target != 'jQuery')
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: "jquery" is not a valid type name

not a valid type name

(lint/suspicious/useValidTypeof)